index.html

```diff
@@ -19,5 +19,8 @@ <h1>Whatever I Want</h1>
     <p class="block">anything</p>
     <button id="dabutton">what the button says</button>
     <div id="counter"></div>
+    <video width="100%" autoplay>
```

Comment:
Though this works, we want to lean towards using CSS instead of inline styles. You can add a class to this tag, and select that class in our `styles.css` and add the `width: 100%` style there

Reply:
The requested changes have been made and pushed